package-lock.json

```diff
@@ -2138,32 +2051,6 @@
                 "inherits": "^2.0.1",

```

Comment:
Github won't let me comment on it because it's so far from the nearest change but there's a dependency on diff 3.3.1 on line 1972, is that going to be an issue?

Reply:
Good catch. I'm just going to force the diff version for now, it looks like its tslint that is pulling in the lower version even though it should be fine with 3.5.0. Probably due to tslint being deprecated soon for eslint (https://github.com/palantir/tslint/issues/4534), so we will need to convert at some point.